explainability-service/src/main/java/org/kie/trustyai/service/prometheus/PrometheusPublisher.java

```diff
@@ -35,10 +36,22 @@ public PrometheusPublisher(MeterRegistry registry) {
         this.values = new HashMap<>();
     }
 
+
     private void createOrUpdateGauge(String name, Iterable<Tag> tags, UUID id) {
         Gauge.builder(name, new AtomicDouble(), value -> values.get(id).doubleValue())
                 .tags(tags).strongReference(true).register(registry);
+    }
 
+    public void removeGauge(String name, UUID id){
+        List<Gauge> gaugesToDelete = registry.get("trustyai_"+name.toLowerCase())
```

Comment:
Since we're using the prefix `trusty_ai` in multiple places now, I would extract this as a constant.

Reply:
done